controllers/comments.js

```diff
@@ -0,0 +1,16 @@
+const { fetchCommentsByArticleId } = require('../models/comments');
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const articleId = req.params.article_id;
+  if (isNaN(articleId)) {
```

Comment:
Nice logic to create a custom error message, but remember that PSQL will give us an error message for this automatically without having to add extra logic